docOps/docs/libraries/nodejs/api_reference.md

```diff
@@ -357,9 +364,9 @@ Synchronizes the account with the Tangle.
 | [options.addressIndex] | <code>number</code> | <code>latest address index</code> | The index of the first address to sync |
 | [options.gapLimit]     | <code>number</code> | <code>10</code>                   | The number of addresses to check       |
 
-**Returns** a [SyncedAccount](#syncedaccount) instance.
+Returns: a [SyncedAccount](#syncedaccount) instance.
 
-#### send(address, amount[, options])
+### send(address, amount[, options])
```

Comment:
### getNodeInfo is also missing, but that's our fault because we didn't add this back then

Could you also add it like this? Otherwise we can do it in a separate PR

```
#### getNodeInfo([url, auth]): NodeInfoWrapper

Gets information about the node.

| Param | Type                              | Default                                                         | Description                    |
| ------- | ------------------------- | ---------------------------------------------- | -------------------------- |
| url       | <code>string</code> | <code>Node from client options</code> | The node url                  |
| auth    | <code>Auth</code>  | <code>undefined</code>                          | The node auth options |

**Returns** the [NodeInfoWrapper](#nodeinfowrapper)
```

where NodeInfoWrapper is 
```JS
interface NodeInfoWrapper {
  url: string
  nodeinfo: NodeInfo
}

interface NodeInfo {
  name: string
  version: string
  isHealthy: boolean
  networkId: string
  bech32HRP: string
  minPoWScore: number
  messagesPerSecond: number
  referencedMessagesPerSecond: number
  referencedRate: number
  latestMilestoneTimestamp: number
  latestMilestoneIndex: number
  confirmedMilestoneIndex: number
  pruningIndex: number
  features: string[]
}
```

Reply:
No problem at all.  I've added the tables, but I would need some more information about the interfaces params, more specifically default values and a short description.   Is there anyone I could ask for this?

Reply:
The values are all set by the nodes, for example like here https://api.hornet-0.testnet.chrysalis2.com/api/v1/info